packages/editor-ui/src/components/NodeDetailsView.vue

```diff
@@ -352,7 +352,9 @@ export default mixins(
 				this.avgOutputRowHeight = 0;
 				this.avgInputRowHeight = 0;
 
-				this.$store.commit('ui/setNDVSessionId');
+				setTimeout(() => {
```

Comment:
Same as comment above about the setTimeout vs $nextTick, here it would be it even simpler: `this.$nextTick(() => this.$store.commit('ui/setNDVSessionId')`